test/index.js

```diff
@@ -889,29 +1234,61 @@ describe('def', function() {
            errorEq(TypeError,
-                   '‘div’ expected a value of type NonZeroInteger ' +
-                   'as its second argument; received 0'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'div :: ValidNumber -> NonZeroInteger -> ValidNumber\n' +
+                   '                      ^^^^^^^^^^^^^^\n' +
+                   '                            1\n' +
+                   '\n' +
+                   '1)  0 :: Number, ValidNumber\n' +
+                   '\n' +
+                   'The value at position 1 is not a member of ' +
+                   'NonZeroInteger.\n'));
 
     throws(function() { div(3, new Number(0)); },
            errorEq(TypeError,
-                   '‘div’ expected a value of type NonZeroInteger ' +
-                   'as its second argument; received new Number(0)'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'div :: ValidNumber -> NonZeroInteger -> ValidNumber\n' +
+                   '                      ^^^^^^^^^^^^^^\n' +
+                   '                            1\n' +
+                   '\n' +
+                   '1)  new Number(0) :: Number, ValidNumber\n' +
+                   '\n' +
+                   'The value at position 1 is not a member of ' +
+                   'NonZeroInteger.\n'));
 
     throws(function() { div(3, -0); },
            errorEq(TypeError,
-                   '‘div’ expected a value of type NonZeroInteger ' +
-                   'as its second argument; received -0'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'div :: ValidNumber -> NonZeroInteger -> ValidNumber\n' +
+                   '                      ^^^^^^^^^^^^^^\n' +
+                   '                            1\n' +
+                   '\n' +
+                   '1)  -0 :: Number, ValidNumber\n' +
+                   '\n' +
+                   'The value at position 1 is not a member of ' +
+                   'NonZeroInteger.\n'));
 
     throws(function() { div(3, new Number(-0)); },
            errorEq(TypeError,
-                   '‘div’ expected a value of type NonZeroInteger ' +
-                   'as its second argument; received new Number(-0)'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'div :: ValidNumber -> NonZeroInteger -> ValidNumber\n' +
+                   '                      ^^^^^^^^^^^^^^\n' +
+                   '                            1\n' +
+                   '\n' +
+                   '1)  new Number(-0) :: Number, ValidNumber\n' +
+                   '\n' +
+                   'The value at position 1 is not a member of ' +
+                   'NonZeroInteger.\n'));
 
     eq(div(4, 2), 2);
     eq(div(4, new Number(2)), 2);
   });
 
-  it('supports the "PositiveInteger" type', function() {
+  it.only('supports the "PositiveInteger" type', function() {
```

Comment:
Not quite done with all the error messages yet, was using `.only` to process a block at a time.